app/views/accounts/index.html.haml

```diff
@@ -23,14 +23,17 @@
       - @account_users.each do |au|
         - account = au.account
         %tr
-          %td= account.account_number_to_s
+          - if au.can_administer? || session_user.administrator?
```

Comment:
Is it possible to use a cancan ability check here rather than an explicit check? I see that's what we use below, so if it's too much effort to figure out, it's fine as is.

Reply:
I'd rather not change this for now if that works, just to make sure we aren't changing the existing behavior.